admin/validate_horace.m

```diff
@@ -114,6 +114,10 @@
     hec.log_level = -1; % turn off informational output
 end
 
+hec

```

Comment:
Short comment on why these are here? Use `disp` to explicitly print (Personally, I find just missing semi-colon non-obvious given half a dozen languages have optional semi-colons)

Reply:
These are just here temporarily for debugging purposes. The builders keep timing out on different tests, and I'm trying to replicate the errors. No luck so far though.

I could add `disp` if we think it's useful to print the config at the start of each test folder run?